### PR TITLE
chore(checkbox): remove dark theme

### PR DESCRIPTION
**Pull request by moog16**

### CODE REVIEW

File: demos/snackbar.html [hunk 1]
```diff
@@ -33,6 +33,14 @@
         background-color: #333;
       }
 
+      .mdc-theme--dark .mdc-checkbox .mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate) ~ .mdc-checkbox__background {
```
Comment (moog16) [inline demos/snackbar.html]: Should I leave as is? Or use sass mixins in the demo.scss file?

Comment (lynnjepsen) [reply demos/snackbar.html]: Remove this. There should be no reference to `mdc-theme--dark` and `mdc-checkbox`

Comment (lynnjepsen) [reply demos/snackbar.html]: In demos/snackbar.scss could you add a call to the checkbox mixins?

Comment (kfranqueiro) [reply demos/snackbar.html]: Given that this is in the snackbar demo html and not the checkbox styles themselves, would it make more sense to leave this for when we remove dark mode from snackbar instead? No sense creating extra work that we're going to remove...

Comment (moog16) [reply demos/snackbar.html]: Yup, spoke to lynn but didn't comment here. I am going to merge in snackbar first, then remove these before merging.

Comment (acdvorak) [reply demos/snackbar.html]: PR #2100 removes dark theme from snackbar, so does it really matter if we break this for like 1 day?

I vote we revert these two rules and leave it broken.

Comment (kfranqueiro) [reply demos/snackbar.html]: I think that's what Matt is saying he wants to do, though he was thinking we could just merge the snackbar PR first and not break anything.

Comment (acdvorak) [reply demos/snackbar.html]: Gotcha. SGTM!

Comment (acdvorak) [inline demos/snackbar.html]: Now that #2100 is merged, we can remove these two `mdc-theme--dark` CSS rules! 😀

**Pull request merged**